Sourcery/Generating/Generator.swift

```diff
@@ -19,6 +19,13 @@ private class TypesReflectionBox: NSObject {
 
     init(types: [Type]) {
         self.types = types
+        
+        for type in types {
```

Comment:
pass in `typesByName` then you can get rid of inner loop for filtering types and it will be faster :)